starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -781,23 +797,24 @@ sub standard_dither{
     my %standard_z_dither = (20 => 768.6,
                              8 => 707.1);
 
+    my $z_amp = int($dthr->{ampl_p} + .5);
```

Comment:
0.5 please.

I also just googled for sec to see if perl has a built-in round() function.  Apparently not.  Wow.